src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -478,14 +474,30 @@ def fuzz(self,
     """LibFuzzerCommon.fuzz override."""
     self._test_ssh()
 
-    #TODO(flowerhack): Pass libfuzzer args (additional_args) here
+    logs.log('SSH works; sync corpus directories')
+    # Push corpus directories to the device.
+    data_dst = self.fuzzer.data_path('corpus')
+    for corpus in corpus_directories:
+      for corpus_dir in os.listdir(corpus):
```

Comment:
why is this "corpus_dir"? each corpus should contain corpus items (files), rather than more directories?

Reply:
because i got my variable names flipped around in my head, whups, fixed to be more clear